python/tests/expressions/test_visitors.py

```diff
@@ -64,15 +64,12 @@
     BooleanExpressionVisitor,
     BoundBooleanExpressionVisitor,
     _ManifestEvalVisitor,
-    inclusive_projection,
```

Comment:
I moved the projection tests into a separate file. I think that makes tests easier to find and work with because you don't have to find the tests in a huge file with lots of unrelated tests.

Reply:
Good call!